composer.json

```diff
@@ -15,7 +15,8 @@
         "drupal/migrate_plus": "^4.0",
         "drupal/migrate_tools": "^4.0",
         "drupal/simplesamlphp_auth": "^3.0",
-        "drupal/jsonapi_extras": "1.x-dev#1a57e11212224129ddb0229726dc7b852bcd836c"
+        "drupal/jsonapi_extras": "1.x-dev#1a57e11212224129ddb0229726dc7b852bcd836c",
+        "drupal/maillog": "1.x-dev#14fd24cd89e299ad271608cd52706c60b753c26d"
```

Comment:
Is this module just helpful for development/debugging purposes, or do we need it for some functionality? I'm inclined to say let's get rid of this comment unless there's a need for it.

If there is a need, let's commit the `composer.lock` file in an additional commit as well.